CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [13.0.0] - 2021-07-12
+### Changed
+- **BREAKING**: Removes AssetsController and adds CollectiblesController and TokensController in its place ([#518](https://github.com/MetaMask/controllers/pull/518))
```

Comment:
Nit: changed sentence to be imperative rather than past-tense, to match the rest of the changelog
```suggestion
- **BREAKING**: Remove AssetsController and add CollectiblesController and TokensController in its place ([#518](https://github.com/MetaMask/controllers/pull/518))
```